src/iGame.cd

```diff
@@ -1,3 +1,5 @@
+#language english
```

Comment:
@coldacid should this CD file be generated automatically by the code, having all the latest updates of the strings?

Reply:
It's the other way around, the code is generated from the CD file.

Reply:
You are right... Thanks for that.